drivers/sensor/bosch/bmp180/bmp180.c

```diff
@@ -327,7 +327,7 @@ static uint32_t bmp180_compensate_press(struct bmp180_data *data)
 	partial_X1 = (cal->ac3 * partial_B6) / 0x2000;
 	partial_X2 = cal->b1 * partial_B6 * (float)(1.0f * partial_B6 / 0x8000000);
 	partial_X3 = (partial_X1 + partial_X2 + 2) / 4;
-	partial_B4 = (uint64_t)(cal->ac4 * (partial_X3 + 32768)) / 0x8000;
+	partial_B4 = (uint64_t)(cal->ac4 * ((int64_t)(partial_X3) + 32768)) >> 15;
```

Comment:
Kinda odd to have the shifting but everywhere else to still perform division. Just a thought :^)